web-registry/src/lib/clients/reactQueryClient.ts

```diff
@@ -4,6 +4,8 @@ export const reactQueryClient = new QueryClient({
   defaultOptions: {
     queries: {
       refetchOnWindowFocus: false,
+      refetchOnMount: false,
+      staleTime: Infinity,
```

Comment:
`staleTime: Infinity` means that we never consider cache stale unless we invalidate part of it ourselves.
Note this is true only for one session, if a user refreshes their browser the cache is getting reset.
Currently, the only data we want to keep up to date during a session are sell orders. For those, cache invalidation happens in a refetch callback.

Reply:
> For those, cache invalidation happens in a refetch callback.

could you point out where that happens so we can see an example? thanks ahead 🙏

Reply:
https://github.com/regen-network/regen-web/pull/1611/files#diff-290478b41c2ea14cca10d871fa14a00a91187ecaa2b2b31689e58b13a4cc2071R40

Reply:
I also added a comment with some additional explanations.